### PR TITLE
feat: Bump cf-deployment to 13.9.0, cats to 6.0.0

### PR DESCRIPTION
**Pull request by viccuad**
## Motivation and Context

Closes: https://github.com/cloudfoundry-incubator/kubecf/issues/941

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code has security implications.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by mook-as** [comment]
Rebased and updated (to new stemcell, etc.).  Passes smoke tests locally, have not tried 🐱 yet.

**Comment by mook-as** [comment]
I believe the only thing remaining for :cat: is that `cf push --strategy=rolling` is broken — it never finishes deploying.

**Comment by mook-as** [comment]
With the last commit I have passing 🐱
```
Ran 98 of 211 Specs in 2008.020 seconds
SUCCESS! -- 98 Passed | 0 Failed | 0 Flaked | 3 Pending | 110 Skipped
```

File: README.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ At the time of this writing these were:
 |kind          | 0.6.0             |                                    |
 |kubectl       | 1.15.6            |                                    |
 |Helm          | 3.0.3             |                                    |
-|CF Operator   | 2.0.0-0.g0142d1e9 |                                    |
-|cf-deployment | 12.18.0           |                                    |
+|CF Operator   | 5.0.0+0.gd7ac12bc |                                    |
```
Comment (viovanov) [inline README.md:58]: should be `5.2.0+0.g22cafc90`

**Comment by viovanov** [comment]
@viccuad can you resolve the conflict please?

**Comment by viccuad** [comment]
@viovanov done

**Comment by mook-as** [comment]
Things seem to be failing with
> Back-off pulling image `docker.io/cfcontainerization/silk:SLE_15_SP1-27.1-7.0.0_374.gb8e8e6af-2.31.0`

Not sure why we're pulling in silk at all.

**Comment by mook-as** [comment]
Okay, we're pulling in silk because #1187 enabled it.  So now I'm really confused: how are we pulling that in, given that [this branch](https://github.com/cloudfoundry-incubator/kubecf/blob/bump-cf-13.9/deploy/helm/kubecf/assets/operations/instance_groups/diego-api.yaml) doesn't have it‽

**Pull request merged**